### PR TITLE
Create an-idea-a-week.md

### PR DESCRIPTION
**Pull request by tomsabin**

### CODE REVIEW

File: source/blog/2015/an-idea-a-week.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+Our people get 10 innovation days a year that are for booking time out from project work. To bolster our innovation strategy and encourage generation and sharing of new ideas we are trial allowing our innovation days to be used to work on people's own ideas. So, if our people aren’t on project work or taking part in some other structured activity, they can use that time to do what they think is valuable.
+
+In order to share what is happening, we have various channels for communicating movement on ideas. There is a monthly rota for Linking and Thinking, a trello board and slack channel. We've also added an Ideas section to Whines and Cheers (our weekly company wide retrospective) and send updates out in our weekly newsletter.
+
+Our team on the bench have daily stand ups to discuss new ideas. This week we are doing a brain dump and tomorrow we hope to plan our collaboration to build an MVP.
+
+Perhaps I just have an annoying habit of labelling things with stupid names.
```
Comment (tomsabin) [inline source/blog/2015/an-idea-a-week.md]: How is this relevant?

Comment (chriscarter90) [inline source/blog/2015/an-idea-a-week.md]: :cactus:
